problem-15/README.md

```diff
@@ -0,0 +1,13 @@
+# Task 15: Reactive form validation (lv2)
+
+## Description
+In the 'ReactiveFormComponent', two validators to the name form control. The validators should ensure that the 
```

Comment:
if you read the desc again, it has no meaning

Reply:
Desc has been updated